src/app/healthyrelationships/healthyrelationships.page.html

```diff
@@ -23,137 +23,134 @@
 
   <div *ngIf="subTab==='healthy'">
     <h1 style="font-size: 18px;">
-      Healthy, whether it's friendship or romantic
+      Signs of a Healthy Relationship
     </h1>
+    <h3 style="text-align: center; font-size: 14px;">
+      (friendship or romantic)
+    </h3>
     <ion-slides pager="true" [options]="slideOpts">
       <ion-slide>
-        <ion-card>
-          <img
-            src="https://static1.squarespace.com/static/58ebeae859cc68b20f410b26/5a281fa5e2c4838eb6e38450/5a302bb5419202a08e5e0347/1513106439847/listen.jpg?format=1500w"
```

Comment:
You have some images local and some images you reference externally. In general, do one or the other. And if you reference externally, you usually want to put all your images on your own CDN, not someone else's. When you reference someone else's image, they can always just remove that URL or block you.